chipy_org/apps/meetings/admin.py

```diff
@@ -57,7 +57,7 @@ class Meta:
 
 
 class MeetingAdmin(admin.ModelAdmin):
-    list_display = ['when', 'where', 'created', 'modified', 'action', 'meeting_type']
+    list_display = ['when', 'where', 'created', 'modified', 'action', 'meeting_type', 'meeting_title']
     list_filter = ['meeting_type']
```

Comment:
@sophm256 could I ask you to take a slightly different approach with this?   As implemented here the use of meeting title and meeting subgroup are getting mixed which causes several if-thens to be used in the templates.  

Instead could you add a `defaul_title` field to the meeting subgroup object?  Then in the template use the meeting title if it's available or fall back to the subgroup default title if available?

Tests would be appreciated (not required) and a good exercise if you haven't done one before.  The pytest style of a function starting with `def test_` is preferred.  Look at my recent bug fix for the meeting RSVP emails if you want an example.  

Keep up the great work.

Reply:
@elmq0022 Thanks for your feedback. **Can you say a bit more about what you mean when you say**

>  As implemented here the use of meeting title and meeting subgroup are getting mixed which causes several if-thens to be used in the templates.

So for the main meeting, the new code doesn't affect the main meeting.
![sample-1](https://user-images.githubusercontent.com/53249319/73188727-ecf08f00-40e8-11ea-8d34-df4568e7ac96.JPG)

Here is what the new code looks like for 3 cases:
![sample-2](https://user-images.githubusercontent.com/53249319/73188717-e82bdb00-40e8-11ea-8f48-a06f49f88c9a.JPG)

Case 1 (Left box) : The  'Algo Sig' is chosen as a 'meeting type' from the drop down menu
Case 2 (middle box): 'Meeting-Title' is chosen from drop down menu 'meeting type'. Then 'meeting title' field is filled with "Future of Robotics"
Case 3 (right box): 'Recursive Sig' is chosen as 'meeting type'. 'My 2nd sub group' is linked to the meeting type 'Recursive Sig'.

I use elif clauses in the template so only the regular meeting types show up or a custom meeting title shows up.  It's an 'exclusive or' relationship.

> Instead could you add a defaul_title field to the meeting subgroup object? Then in the template use the meeting title if it's available or fall back to the subgroup default title if available?

![meeting](https://user-images.githubusercontent.com/53249319/73191456-25926780-40ed-11ea-9207-91fe5fca8cb9.JPG)

If you look at the pic above, the 'meeting type' looks like 'Sig Meeting' or 'Mentorship Meeting'. This is what usually shows up as the title of the event. The meeting subgroup is an optional object and sometimes doesn't show up on the template/homepage. This is because a meeting subgroup is an optional object to be associated with the meeting type.  In my new code and the original code, the meeting subgroup's name shows up as a subtitle to the 'meeting type' when the 'meeting type' is the title of the event (which was case 3 in the picture earlier). (Below are pictures of a subgroup and a meeting type in the admin panel.)

![meeting-type2](https://user-images.githubusercontent.com/53249319/73191516-393dce00-40ed-11ea-86a7-43a7e6df1291.JPG)
![subgroup](https://user-images.githubusercontent.com/53249319/73191547-45c22680-40ed-11ea-9d83-35bfc100218e.JPG)

I'm cool with implementing it how you want me to, but I wasn't sure if we were in agreement with what was going on. Then again, I may be the one that's confused :) 

Since you have access to the database, could you please give me a concrete example of what a 'meeting type' name is and what a 'subgroup' name is? Thanks. I'm unclear what is the difference between the two.

I appreciate your help and guidance on how to improve things!